packages/driver/test/cypress/integration/commands/actions/check_spec.js

```diff
@@ -0,0 +1,1233 @@
+const $ = Cypress.$.bind(Cypress)
+const {
+  _,
+} = Cypress
+const {
+  Promise,
+} = Cypress
```

Comment:
These are pulling from the same source, can join them.

```js
const { _, Promise } = Cypress
```

Reply:
Agree. Will fix.